app/internal/page/address_form.go

```diff
@@ -63,6 +63,8 @@ func (f *addressForm) Validate() validation.List {
 			validation.StringTooLong(50))
 		errors.String("address-town", "townOrCity", f.Address.TownOrCity,
 			validation.Empty())
+		errors.String("address-postcode", "aPostcode", f.Address.Postcode,
+			validation.Empty())
```

Comment:
I think the reason this was originally left out was because we need to allow international addresses (although I imagine that may need other changes too, so happy for it be added in on this I guess)